src/lib/updates/avatar.js

```diff
@@ -24,11 +24,11 @@ const uploadAvatars = async (lastUpdate, prevVersion, log) => {
   } else {
     // if already cid - check is cid valid and exists, set null (delete avatar) if fails
     try {
-      const base64 = await userStorage.loadAvatar(avatar, true)
-
-      if (!base64) {
+      if (!isValidCIDImage(avatar)) {
         throw new Error('Not a valid CID')
       }
+
+      await userStorage.loadAvatar(avatar, true)
```

Comment:
load avatar was removed

Reply:
yep, i changed it but have forgot to push